dashboard/test/controllers/survey_results_controller_test.rb

```diff
@@ -0,0 +1,18 @@
+require 'test_helper'
+
+class SurveyResultsControllerTest < ActionController::TestCase
+
+  setup do
+    @teacher = create(:teacher)
+  end
+
+  test "post survey results" do
+    sign_in @teacher
+    post :create, {survey: {survey2016_ethnicity_asian: 22}, format: :json}
```

Comment:
you can also use assert_creates here (which would additionally assert that this is not creating multiple surveyresults)